src/Youtube.php

```diff
@@ -250,6 +250,28 @@ public function searchChannelVideos($q, $channelId, $maxResults = 10, $order = n
         return $this->searchAdvanced($params, $pageInfo);
     }
 
+    
+    /**
+     * @param $PlaylistId
+     */
+    public function getAllPlaylistItemsByPlaylistId($playlistId)
```

Comment:
@badrshs This can be changed to just getPlaylistItems since it doesn't return all playlist items, and handle the pagination separately